From 9b97d734042b561c3560c1414f5f14eca4a91844 Mon Sep 17 00:00:00 2001
From: shahed <jamshidalinezhadian@gmail.com>
Date: Mon, 12 Oct 2026 00:21:47 +0330
Subject: [PATCH 3/3] post_proc: include stdio.h for snprintf

Android 14 builds vendor modules against liblog's include_vndk log.h, which no longer includes <stdio.h>. effect_api.c calls snprintf and relied on that transitive include.

Change-Id: I4f92e01c186740b47736d17b8600acf0a99ca192
---
 post_proc/effect_api.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/post_proc/effect_api.c b/post_proc/effect_api.c
index 7aef997..4096609 100644
--- a/post_proc/effect_api.c
+++ b/post_proc/effect_api.c
@@ -55,6 +55,7 @@
 #endif
 
 #include <stdbool.h>
+#include <stdio.h>
 #include <cutils/log.h>
 #include <tinyalsa/asoundlib.h>
 #include <sound/audio_effects.h>
-- 
2.43.0

